extensions/amp-story-auto-ads/0.1/story-ad-localization.js

```diff
@@ -95,7 +95,9 @@ export class StoryAdLocalization {
    * Create localization service and register all bundles.
    */
   init_() {
-    this.localizationService_ = new LocalizationService(this.win_);
+    this.localizationService_ = Services.localizationForDoc(
+      this.win_.document.body
```

Comment:
Can stories displayed in shadow/PWA mode? If so, this won't work.

Reply:
Nope, stories aren't made for these environments. /cc @newmuis

Reply:
Heh, the main purpose of "forDoc" vs. "window" services is to support shadow/PWA. So if that was always out of scope, we technically didn't need this change.

Reply:
💀